docker-registry/Dockerfile

```diff
@@ -4,7 +4,7 @@ ENV SETTINGS_FLAVOR s3
 ENV STORAGE_PATH /registry
 ENV SEARCH_BACKEND sqlalchemy
 ENV AWS_ENCRYPT false
-ENV AWS_DEBUG true 
+ENV AWS_DEBUG true
```

Comment:
Never mind; I just noticed the difference was not boolean, but a stray trailing space